deployment/go.mod

```diff
@@ -10,6 +10,7 @@ replace github.com/smartcontractkit/chainlink/v2 => ../
 require github.com/smartcontractkit/chainlink/v2 v2.27.3-0.20250908153844-03478edcd69f
 
 require (
+	cosmossdk.io/errors v1.0.1
```

Comment:
hmmm did the IDE pull it in automatically instead of your favorite `github.com/pkg/errors`?

Reply:
No go fmt moved this around